system-x/services/db/mysql/src/main/java/software/tnb/db/mysql/service/MySQL.java

```diff
@@ -37,10 +37,15 @@ public Map<String, String> containerEnvironment() {
         );
     }
 
-    public SQLValidation rootValidation() {
+    public synchronized SQLValidation rootValidation() {
```

Comment:
why this needs to be synchronized?

Reply:
to avoid multiple access, since there is a check on the variable

Reply:
can you elaborate please? is there a case where there are multiple threads using this?

Reply:
mmm, you are right, my bad, I considered the variable `rootValidation` as static but it is not, so I revert this line